src/utils/units.py

```diff
@@ -5,6 +5,12 @@ class Units:
     units_length = {"Km": "Kilometres",
             "M":"Meter",
             "c":"Centimeter"}
```

Comment:
"m" instead of "M"
"km" instaed of "Km"
"cm" instead of "c"

Add:
"ft": "feet"
"mi": "international mile"
"yd": "international yard"